docs_outdoornav_user_manual/getting_started/tf_validation.mdx

```diff
@@ -0,0 +1,73 @@
+---
+title: TF Validation
+sidebar_label: TF Validation
+sidebar_position: 4
+toc_min_heading_level: 2
+toc_max_heading_level: 4
+---
+
+:::note
+
+Sensor TF validation should only be required if the performance is

```

Comment:
Can we write out what "TF" means or provide a link to the corresponding ROS documentation?